integration-cli/docker_api_inspect_test.go

```diff
@@ -42,3 +43,41 @@ func (s *DockerSuite) TestInspectApiContainerResponse(c *check.C) {
 		c.Fatalf("Path of `true` should not be converted to boolean `true` via JSON marshalling")
 	}
 }
+
+func (s *DockerSuite) TestInspectApiImageResponse(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "pull", "busybox")
+	out, _, err := runCommandWithOutput(runCmd)
+
+	if err != nil {
+		c.Fatalf("failed to pull image: %s, %v", out, err)
+	}
+
+	endpoint := "/images/busybox/json"
+	status, body, err := sockRequest("GET", endpoint, nil)
+
+	c.Assert(err, check.IsNil)
+	c.Assert(status, check.Equals, http.StatusOK)
+
+	var inspectJSON types.ImageInspect
+	if err = json.Unmarshal(body, &inspectJSON); err != nil {
+		c.Fatalf("unable to unmarshal body for latest version: %v", err)
+	}
+
+	c.Assert(len(inspectJSON.Tags), check.Equals, 1)
+	c.Assert(inspectJSON.Tags[0], check.Equals, "busybox:latest")
+
+	runCmd = exec.Command(dockerBinary, "tag", "busybox:latest", "busybox:mytag")
```

Comment:
please use `dockerCmd(c, "tag", "busybox:latest", "busybox:mytag")`, which automatically checks/fails the test if there is an error.